src/backend/utils/base-decorator.js

```diff
@@ -119,22 +125,22 @@ class BaseDecorator {
   /**
    * Returns object(map) with default actions as keys and their values
    */
-  getDefaultActions(helpers, record) {
+  getDefaultActions(record) {
     const resource = this._resource
     return {
       show: {
-        path: helpers.showRecordUrl(resource, record),
-        icon: 'info',
+        path: this.helpers.showRecordUrl(resource, record),
+        icon: 'fas fa-info',
```

Comment:
I changed it in a way that all classes are given here -  did that because user can add class from other library (not fa)